lib/Runtime/Base/FunctionBody.cpp

```diff
@@ -3635,21 +3811,22 @@ namespace Js
 
     void FunctionBody::SetStackNestedFuncParent(FunctionBody * parentFunctionBody)
     {
-        Assert(this->stackNestedFuncParent == nullptr);
+        Assert(this->get_AuxPtr(e_stackNestedFuncParent) == nullptr);
```

Comment:
get_AuxPtr/set_AuxPtr are special here. other Get*/Set* wraps these two to do the real access. but for this particular one, should use GetStackNestedFuncParent() instead. I'll fix this